modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -109,20 +109,25 @@ export default class Tile3DLayer extends CompositeLayer {
   }
 
   async _loadTileset(tilesetUrl) {
-    const {loadOptions} = this.props;
+    const {loadOptions = {}} = this.props;
 
     // TODO: deprecate `loader` in v9.0
     let loader = this.props.loader || this.props.loaders;
     if (Array.isArray(loader)) {
       loader = loader[0];
     }
 
-    const options = {...loadOptions};
+    const options = {loadOptions};
```

Comment:
Recommend avoid mutating the user provided `loadOptions` object

Reply:
Done